Google.Api.Generator/CodeGenerator.cs

```diff
@@ -238,6 +242,15 @@ internal static class CodeGenerator
                 var snippetsCsprojContent = CsProjGenerator.GenerateSnippets(ns);
                 var snippetsCsProjFilename = $"{snippetsPathPrefix}{ns}.Snippets.csproj";
                 yield return new ResultFile(snippetsCsProjFilename, snippetsCsprojContent);
+                // Generate snippet metadata (only for standalone snippets).
+                // If we have content, then the last service on the list is from the same package (both proto and C#
```

Comment:
I'm confused as to why we're using last rather than first here... but then I don't know much about the context.

Reply:
I still don't get why we want to use Last rather than First here (as we do elsewhere). But I'm happy for you to submit as-is and explain it to me offline - I can then suggest a clearer comment, potentially.

Reply:
Added the changes that we agreed on offline on the last commit.